src/Main/BackToTop/BackToTop.module.scss

```diff
@@ -41,6 +42,10 @@ $back-to-top-height: rem(
       transform: translateY(-#{rem($nds-spacing-xx-small)});
     }
   }
+
+  .container {
+    display: flex;
```

Comment:
I think we should probably have `align-items: center;` here too to makes sure the svg icon always aligns with the middle of the text